src/app/profile-page/profile-page.component.ts

```diff
@@ -163,4 +171,9 @@ export class ProfilePageComponent implements OnInit {
   submit() {
     this.updateProfile();
   }
+
+  isResearcherProfileEnabled(){
```

Comment:
This new method has no Specs.  We should add basic specs to prove it's returning true/false when expected.